tasks/interactive-bots.md

```diff
@@ -64,7 +64,7 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
   should point to your development environment URL.
 
   If you're running your bot in the droplet, alongside your Zulip instance, the
-  `<dev-url>` will be `localhost:9991`.
+  `<dev-url>` will be `localhost:9991` or `<username>.zulipdev.org:99991`.
```

Comment:
The `<username>.zulipdev.org:99991` was not working for a lot of students - did it work for you? I've written `localhost:9991` here purposefully and more students seem to not have problems with running the bot in the droplet.